web/gui/dashboard_info.js

```diff
@@ -3825,10 +3825,10 @@ netdataDashboard.context = {
         info: '<p><b>Connections usage</b> across all databases. The maximum number of concurrent connections to the database server is (<i>max_connections</i> - <i>superuser_reserved_connections</i>). As a general rule, if you need more than 200 connections it is advisable to use connection pooling.</p><p><b>Available</b> - new connections allowed. <b>Used</b> - connections currently in use.</p>'
     },
     'postgres.checkpoints': {
-        info: '<p>Number of checkpoints that have been performed. Checkpoints are periodic maintenance operations the database performs to make sure that everything it’s been caching in memory has been synchronized with the disk. It’s desirable when checkpoints are scheduled rather than requested, as the latter can indicate that your databases are under heavy load.</p><p><b>Scheduled</b> - checkpoints triggered due that the time elapsed from the previous checkpoint is more than pg setting <i>checkpoint_timeout</i>. <b>Requested</b> - checkpoints ran due to uncheckpointed WAL size grew to more than <i>max_wal_size</i> setting.</p>'
+        info: '<p>Number of checkpoints that have been performed. Checkpoints are periodic maintenance operations the database performs to make sure that everything it’s been caching in memory has been synchronized with the disk. This is essential in case of failure scenarios as the standby server will know where to start replaying transactions from.</p><p>A high number of checkpoints being requested may indicate your databases are under heavy load. Increasing <a href="https://www.postgresql.org/docs/current/runtime-config-wal.html#GUC-MAX-WAL-SIZE" target="_blank"><i>max_wal_size</i></a> may help checkpoints become time-driven (scheduled) as opposed to load-driven (requested), up to a point.</p><p><b>Scheduled</b> - checkpoints triggered as per schedule when time elapsed from the previous checkpoint is greater than <a href="https://www.postgresql.org/docs/current/runtime-config-wal.html#GUC-CHECKPOINT-TIMEOUT" target="_blank"><i>checkpoint_timeout</i></a>. <b>Requested</b> - checkpoints triggered due to WAL updates reaching the <i>max_wal_size</i> before the <i>checkpoint_timeout</i> is reached.</p>'
```

Comment:
A note: turning dashboard info into a wiki page could be more annoying than helpful for users. It (in theory) sounds nice, but just a wall of text when you check the just regularly.

In addition, I am not sure debugging information (e.g. "tuning A **may** help") belongs to a chart:

- it adds text.
- it describes one or a few possible reasons and misses the rest.
- collides with alerts and alerts guides (I think a dead project, but the idea was to write troubleshooting guides).

Reply:
Fair points - made some updates. How about now?